emcc.py

```diff
@@ -810,6 +810,9 @@ def optimizing(opts):
       # Compiling C code with .c files, don't enforce a default C++ std.
       clang_compiler = CC
 
+    if '-print-search-dirs' in newargs:
+      return run_process([clang_compiler, '-print-search-dirs'], check=False).returncode
```

Comment:
Why can't we just allow this argument to be passed through to clang like others?  i.e. can we avoid the special case here?